GitUI/CommandsDialogs/SettingsDialog/Plugins/PluginSettingsPage.cs

```diff
@@ -58,14 +58,14 @@ private IEnumerable<ISetting> GetSettings()
                 throw new ApplicationException();
             }
 
-            return _gitPlugin.GetSettings();
+            return _gitPlugin.HasSettings ? _gitPlugin.GetSettings() : Enumerable.Empty<ISetting>();
```

Comment:
Will this work?
```suggestion
            return _gitPlugin.HasSettings ? _gitPlugin.GetSettings() : Array.Empty<ISetting>();
```

If I'm not mistaken, in .NET Core these were optimised, in .NET Framework `Enumerable.Empty<T>()` is less efficient than `Array.Empty<T>`

Reply:
yes. done.